.github/workflows/ft-sync.yml

```diff
@@ -9,6 +9,8 @@ jobs:
   everything:
     name: ft-sync
     runs-on: macos-10.15
+    env:
+      FT_AUTH_CODE: abcd
```

Comment:
Nopes, to send ENV you have to update github settings. You cant store the auth code in publicly visible place.